src/flow.h

```diff
@@ -497,6 +498,8 @@ typedef struct Flow_
     uint64_t todstbytecnt;
     uint64_t tosrcbytecnt;
 
+    PcapFileInfo *pcap_info;
```

Comment:
It looks bad to grow this structure, especially for afpacket users who will not need it...